### PR TITLE
Highlights for v5

### PR DESCRIPTION
**Pull request by thomasdax98**

### CODE REVIEW

**Comment by johnnyomair** [comment]
What would be the workflow when releasing a new version? Clear/remove the file in the "Version packages" PR?

**Comment by johnnyomair** [comment]
I believe an advantage of such a file would be that we don't have to manually look for highlights once we plan to release a version. Devs may add their change to HIGHLIGHTS.md themselves (we'll probably have to filter highlights upon release anyway, not everything is a highlight 😁 )

A ideal solution would be if changsets could automatically remove the file and add it to the changelog. We probably would need to write our own GitHub action, but that is a future improvement.

**Comment by thomasdax98** [comment]
For now, I think it's okay that I just write the highlights myself. I checked them into Git mostly for collaboration purposes.

**Comment by johnnyomair** [comment]
> For now, I think it's okay that I just write the highlights myself. I checked them into Git mostly for collaboration purposes.

So we don't add such a file and close this PR once v5 is released? Fine by me

**Comment by thomasdax98** [comment]
Yes, let's do that

**Review by nsams — DISMISSED** [review]

File: HIGHLIGHTS.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+### Dependencies
+
+The new `DependenciesService` provides methods to create and access an index containing all dependencies from blocks to entities.
+
+Note: Currently, the library doesn't offer a way to display the dependencies in the admin. This will be added in a later minor release of v5.
+
+More information in section [@comet/cms-api](#@comet/cms-api).
+
+### No "Discard unsaved Changes" Dialog in Tabs
```
[the remark this answers is not in the document]
Comment (thomasdax98) [reply HIGHLIGHTS.md]: Yes makes sense, I concretized that

**Pull request closed without merging**